.travis.yml

```diff
@@ -6,14 +6,15 @@ go:
   - 1.7
 env:
   global:
-    - LIBRDKAFKA_VERSION=v0.9.2
-    - PKG_CONFIG_PATH="$HOME/gopath/src/github.com/confluentinc/confluent-kafka-go/tmp-build-${LIBRDKAFKA_VERSION}/lib/pkgconfig"
+    - PKG_CONFIG_PATH="$HOME/gopath/src/github.com/confluentinc/confluent-kafka-go/tmp-build/lib/pkgconfig"
     - PATH="$PATH:$GOPATH/bin"
-cache:
-  directories:
-    - tmp-build-${LIBRDKAFKA_VERSION}
+  matrix:
+    - LIBRDKAFKA_VERSION=v0.9.2
+    - LIBRDKAFKA_VERSION=master
+
 before_install:
-  - test -f tmp-build-${LIBRDKAFKA_VERSION}/lib/librdkafka.a || bash mk/bootstrap-librdkafka.sh ${LIBRDKAFKA_VERSION} tmp-build-${LIBRDKAFKA_VERSION}
+  - rm -rf tmp-build
+  - test -f tmp-build/lib/librdkafka.a || bash mk/bootstrap-librdkafka.sh ${LIBRDKAFKA_VERSION} tmp-build
```

Comment:
I guess it doesn't hurt, but given the above `rm -rf tmp-build`, won't the initial `test` here always fail?